app/src/main/java/com/example/boltassist/OverlayService.kt

```
@@ -0,0 +1,171 @@
+        overlayView.addView(startStop)
+
+        val amountTv = TextView(this).apply {
+            setTextColor(ContextCompat.getColor(context, android.R.color.white))
+            text = "0.00 zł"
+        }
+        val plus = Button(this).apply {
+            text = "+5"
+            setOnClickListener {
+                if (isRunning) {
+                    amount += 5f
+                    amountTv.text = String.format("%.2f zł", amount)
```

detekt (1.23.8)

[warning] String.format("%.2f zł", amount) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)